### PR TITLE
Changing Keep specific description of prettier config to be used for Thesis wide projects

### PR DESCRIPTION
**Pull request by dimpar**
This PR:

- Changes README, cleaned up Keep project references and replaced with `Thesis`
- Replaced `npm` with `yarn` because we started using `yarn`
- Updated License and `package.json`

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
 ```json
-"prettier": "@keep-network/prettier-config-keep"
+"prettier": "@thesis/prettier-config"
 ```
 +
 For more sophisticated configuration instead of adding this entry add a configuration
 file (see: <<Plugins>> section).
 
-3. Define `scripts` property in `package.json`:
+3. Define `scripts` property in `package.json`, e.g.:
```
Comment (Shadowfiend) [inline README.adoc:29]: Thoughts on adding a note here that for JavaScript/TypeScript formatting in particular, Thesis projects should use the prettier-eslint integration in `@thesis/eslint-config`?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (Shadowfiend) [inline yarn.lock:4]: Man I freaking love an empty lockfile 🤩

File: LICENSE [hunk 3]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 keep.network
+Copyright (c) 2021 Thesis
```
Comment (Shadowfiend) [inline LICENSE]: Need to double-check--this may need a `, Inc` on the end.

**Review by Shadowfiend — APPROVED** [review]
This looks great. Left a couple of small notes that may be worth fixing before merge.

File: README.adoc (hunk 1 above)
Comment (dimpar) [reply README.adoc:29]: Sure thing 5e70902

File: LICENSE (hunk 3 above)
Comment (dimpar) [reply LICENSE]: Matt confirmed that `Thesis, Inc` is a preferred one 037b2c5

**Review by Shadowfiend — APPROVED** [review]
🚢

**Pull request merged**